install/jakartaee/bin/sig-test-pkg-list_se8.txt

```diff
@@ -164,19 +164,19 @@ jakarta.xml.bind.annotation.adapters
 jakarta.xml.bind.attachment
 jakarta.xml.bind.helpers
 jakarta.xml.bind.util
-jakarta.xml.registry
-jakarta.xml.registry.infomodel
-jakarta.xml.rpc
-jakarta.xml.rpc.encoding
-jakarta.xml.rpc.handler
-jakarta.xml.rpc.handler.soap
-jakarta.xml.rpc.holders
-jakarta.xml.rpc.server
-jakarta.xml.rpc.soap
+javax.xml.registry
```

Comment:
javax.xml.registry + javax.xml.rpc should be removed as they are pruned in EE 9.

Reply:
Should be removed now